### ISSUE TITLE
WSL not at all working

### ISSUE THREAD

**Issue by VijaySutariaJCD** [microsoft/vscode-remote-release#4383]
Issue Type: <b>Bug</b>

this started happening suddenly , not sure how to reproduce this

Extension version: 0.52.0
VS Code version: Code 1.52.1 (ea3859d4ba2f3e577a159bc91e3074c5d85c0523, 2020-12-16T16:34:46.910Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz (8 x 1800)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.90GB (4.74GB free)|
|Process Argv|--crash-reporter-id a685548b-82d1-401a-b6b9-3b0a175472c8|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
openlogontheside:30221877
python383:30185418
vspyt653:30244269
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30211401
wsl2promptcf:30224613
vstry244:30250195
pythonvsdeb440:30248342
unusedpromptcf:30224611
folderexplorer:30224614
openfilemenu:30224647
pythonvsded773:30248341
pythonvspyt600:30247031

```

</details>

**Comment by aeschli** [comment]
@VijaySutariaJCD Can you give more details? Are you talking about the VSCode-Remote WSL extension?
You open a VSCode WSL window but it fails? What's in the WSL Log?

**Comment by VijaySutariaJCD** [comment]
@aeschli . Its started working fine now after I installed a new Linux Server in my windows machine. 

Issue was: VSCode was showing extension installed and when I was trying to "reopen folder in WSL", It was not doing/showing anything and even it was not enabling WSL. Even the Remote Explorer window was not showing anything and the loading bar was keep rotating. 
I restarted my machine as well as installed new Linux Server and now all good.

Thanks @aeschli
